src/mlpack/tests/main_tests/hoeffding_tree_test.cpp

```diff
@@ -655,28 +640,30 @@ BOOST_AUTO_TEST_CASE(HoeffdingBinarySplittingStrategyTest)
   mlpackMain();
 
   // Check that number of children is 2.
-  BOOST_REQUIRE_EQUAL(
-      (IO::GetParam<HoeffdingTreeModel*>("output_model"))->NumNodes()-1, 2);
+  REQUIRE((IO::GetParam<HoeffdingTreeModel*>("output_model"))->NumNodes()-1
+      == 2);
```

Comment:
```suggestion
  REQUIRE(
      (IO::GetParam<HoeffdingTreeModel*>("output_model"))->NumNodes() - 1 == 2);
```
Tiny style fix (with the spaces around `-1`), and I thought it looked a little better wrapped like this. :smile: